h/static/scripts/filter/test/document-domain-test.js

```diff
@@ -29,6 +29,15 @@ describe('documentDomain', function() {
     assert(domain === '');
   });
 
+  it('returns the filename for local documents with titles', function() {
+    var domain = documentDomainFilterProvider()({
+      title: 'example.com',
+      uri: 'file:///home/seanh/MyFile.pdf'
+    });
+
+    assert(domain === '(MyFile.pdf)');
```

Comment:
Use `assert.equal(actual, expected)` rather than `assert(actual === expected)` and you'll get more helpful error messages when the test fails.